eslint.config.mjs

```diff
@@ -578,33 +578,58 @@ export default tseslint.config(
   },
   {
     extends: [perfectionistPlugin.configs['recommended-alphabetical']],
-    files: ['packages/utils/src/**/*.ts'],
+    ignores: ['packages/eslint-plugin/src/configs/*'],
+    files: [
+      'packages/eslint-plugin/{src,tests,tools,typings}/**/*.ts',
+      'packages/utils/src/**/*.ts',
+    ],
     rules: {
-      'perfectionist/sort-classes': [
+      '@typescript-eslint/sort-type-constituents': 'off',
```

Comment:
Perfectionist's documentation says it can conflict with `sort-intersection-types`, but I never encountered any conflict issue myself.

Reply:
Yeah it's mostly around small nuances in sorting, or when you have custom sorts.